Makefile

```diff
@@ -0,0 +1,47 @@
+BOLD := \033[1m
+RESET := \033[0m
```

Comment:
I don't think this is required...

Reply:
explain ?

Reply:
Ok I see this comment before the one asking to replace the `help`